### PR TITLE
media-gfx/dpic: Add upstream to metadata.xml

### PR DESCRIPTION
**Pull request by jonasstein**

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@jonasstein could you please format the `metadata.xml` using `xmllint`, preferably along the lines of
```
XMLLINT_INDENT="	" xmllint --format
```
where the whitespace character is a tab, not some spaces (insert tabs in the console with Ctrl-V + Tab), and remove empty lines

**Comment by jonasstein** [comment]
@SoapGentoo Thank you for pointing me to that. I should not use different editors without checking, what they do with tabs. Your xmllint line is very handy.

**Comment by SoapGentoo** [comment]
@jonasstein good, but you should prefer proper tabs over spaces in **metadata.xml**. Hence try the Ctrl-V trick to get a tab character into the `XMLLINT_INDENT` variable.

File: media-gfx/dpic/metadata.xml [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+        <doc>https://ece.uwaterloo.ca/~aplevich/dpic/dpicdoc.pdf</doc>
+        <maintainer status="active">
+            <name>J. D. Aplevich</name>
+        </maintainer>
+    </upstream>
+    <maintainer type="project">
+        <email>tex@gentoo.org</email>
+        <name>Gentoo TeX Project</name>
+    </maintainer>
+    <longdescription> Dpic is an implementation of the pic "little language" for creating line drawings and illustrations for documents, web pages, and other uses. </longdescription>
```
Comment (SoapGentoo) [inline media-gfx/dpic/metadata.xml]: Also remove the leading and trailing whitespace

**Comment by jonasstein** [comment]
@SoapGentoo Changed to TAB as you wished. In fact there is no indentation rule for metadata.xml yet. I had recently discussed that on IRC. Unfortunately this is not well documented. See also  https://bugs.gentoo.org/show_bug.cgi?id=591548
But I am quite relaxed and flexible in that case. Learned in the meantime, that several editors destroy the metadata indentation with their "autocorrection". I used now vim, that worked better.

**Comment by SoapGentoo** [comment]
@jonasstein you're absolutely right in that there is no global indentation policy for metadata.xml. I just prefer tabs 😄. Thanks, I'll merge it in a minute.

**Pull request closed without merging**